packages/amplify-graphql-searchable-transformer/src/__tests__/amplify-graphql-searchable-transformer.tests.ts

```diff
@@ -197,15 +197,15 @@ test('it generates expected resources', () => {
       EBSOptions: anything(),
       ElasticsearchClusterConfig: anything(),
       ElasticsearchVersion: '7.10',
-    })
+    }),
   );
   cdkExpect(searchableStack).to(
     haveResource('AWS::AppSync::DataSource', {
       ApiId: {
         Ref: anything(),
       },
       Name: 'ElasticSearchDataSource',
-      Type: 'AMAZON_ELASTICSEARCH',
+      Type: 'AMAZON_OPENSEARCH',
```

Comment:
Does this change work (can you push it to the cloud)? It looks like this data source type is not [documented](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-datasource.html#cfn-appsync-datasource-type).

Reply:
These have been reverted back.